charts/sorry-cypress/templates/deployment-dashboard.yml

```diff
@@ -19,7 +19,7 @@ spec:
       - env:
         {{- range .Values.api.ingress.hosts }}
         - name: GRAPHQL_SCHEMA_URL
-          value: {{ .host }}
+          value: http://{{ .host }}
```

Comment:
Dunno if this is the best practice or if we should add a variable to values just like `ciUrl`, because the ingress host may be `https`, have a port, etc.

Reply:
@tico24, @agoldis, share your thoughts :)

Reply:
Yup, you're right... all the internal chatter will be on 80.

**However** I believe this is a URL that the user's browser connects to, so it needs to be the resulting ingress URL, not the internal one. Even in the docker-compose stack, if you set it to the internal hostname, then the dashboard doesn't load.

Reply:
Alright, then I'm changing it to a custom variable :)